src/components/common/_variables.scss

```diff
@@ -21,3 +21,14 @@
     }
   }
 }
+
```

Comment:
i missed this in earlier reviews but _variables.scss should be for variables only, not common code

for example, see bootstraps; it's only variables:
https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss